calculate_dora_metrics.py

```
@@ -0,0 +1,332 @@
+    with open(file_path, "r", encoding="utf-8") as f:
+        lines = f.readlines()
+
+    # Find table headers
+    header_line = None
+    for i, line in enumerate(lines):
+        if "Run #" in line and "Status" in line:
+            header_line = i
+            break
+
+    if header_line is None:
+        raise ValueError("Could not find table header")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)